go/vt/vtctl/reparentutil/reparent_sorter_test.go

```diff
@@ -93,54 +94,67 @@ func TestReparentSorter(t *testing.T) {
 		tablets          []*topodatapb.Tablet
 		innodbBufferPool []int
 		positions        []replication.Position
+		backingUpTablets map[string]bool
```

Comment:
```suggestion
		tabletBackupStatus map[string]bool
```

Reply:
`backingUpTablets` makes me think it's a collection of tablets. This is not a collection of tablets, it's a kind of tablet state.
`tabletBackupState` or `isTabletRunningBackup` or some such thing will also convey the sense of this map properly.